### DISCUSSION TITLE
Azam sports please

### DISCUSSION THREAD

**Discussion by ahmedzooo** [iptv-org/community#180] [Q&A]
**Channel Name:** Azam sports
**Country:** Tanzania
**Language:** Swahili
**Website:** azamtvmax.com
**Stream URL (optional):** xxx
**Notes (optional):** xxx

**Comment by Dum4G** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/2730

**Comment by ahmedzooo** [comment]
Please i own for account for Azamtvmax.apk

**Comment by Dum4G** [comment]
I can move this issue to Discussions, but I can't guarantee you that we will be able to retrieve links from an app.

**Reply by ahmedzooo** [reply]
https://1521034005.rsc.cdn77.org/1521034005/Manifest.mpd

**Reply by ahmedzooo** [reply]
azamsport 2

**Reply by ahmedzooo** [reply]
Can it be converted into m3u8

**Reply by Dum4G** [reply]
Interesting. They do have a file Manifest.m3u8 and it even has 3 resoulution streams in the old format. However, it loads infinitely for me

#EXT-X-STREAM-INF:CLOSED-CAPTIONS=NONE,RESOLUTION=-1x360,FRAME-RATE=29.970,CODECS="avc1.64001f,mp4a.40.2",AVERAGE-BANDWIDTH=980000,BANDWIDTH=1230000
tracks-v3a1/mono.m3u8
#EXT-X-STREAM-INF:CLOSED-CAPTIONS=NONE,RESOLUTION=-1x480,FRAME-RATE=29.970,CODECS="avc1.64001f,mp4a.40.2",AVERAGE-BANDWIDTH=1750000,BANDWIDTH=2190000
tracks-v2a1/mono.m3u8
#EXT-X-STREAM-INF:CLOSED-CAPTIONS=NONE,RESOLUTION=-1x540,FRAME-RATE=29.970,CODECS="avc1.64001f,mp4a.40.2",AVERAGE-BANDWIDTH=2260000,BANDWIDTH=2830000
tracks-v1a1/mono.m3u8

**Reply by tzniceguy** [reply]
How do you extract the streaming links from the App ?

**Reply by tzniceguy** [reply]
How do you extract the streaming links from the App ?
